yarn.lock

```diff
@@ -59,6 +59,28 @@
     semver "^5.4.1"
     source-map "^0.5.0"
 
+"@babel/core@^7.6.2":
+  version "7.9.0"
```

Comment:
Looks like we added duplicate babel packages in a bunch of places. Could we remove the duplicates? (IIRC Atlassian had a tool to do this globally).